src/pages/Host.tsx

```diff
@@ -124,7 +124,7 @@ export const Host: React.VFC = () => {
             <PageSider width={350}>
               <Metadata
                 loading={hostMetaDataLoading}
-                data={hostData}
+                host={hostData?.host}
```

Comment:
I think we already have this variable in the file

```suggestion
                host={host}
```